eng/stylecop.json

```diff
@@ -0,0 +1,15 @@
+{
+    "$schema": "https://raw.githubusercontent.com/DotNetAnalyzers/StyleCopAnalyzers/master/StyleCop.Analyzers/StyleCop.Analyzers/Settings/stylecop.schema.json",
+    "settings": {
+        "documentationRules": {
+            "copyrightText": "Copyright (C) Microsoft Corporation. All rights reserved.\nLicensed under the MIT License",
+            "companyName": "Microsoft Corporation",
+            "xmlHeader": false,
+            "documentExposedElements": true,
```

Comment:
Why do we need all these documentExposed* parameters?

Reply:
I'd keep the rules here minimal and rely more on .editorconfig. Unfortunately, .editorconfig won't fail the build, though, while this has the potential to if that's what we want (probably eventually).

Reply:
Those serve to enforce XML documentations on Public members Interfaces and Internals types.